### ISSUE TITLE
Engine Android a11y unit test flaking

### ISSUE THREAD

**Issue by zanderso** [flutter/flutter#114036]
From https://ci.chromium.org/ui/p/flutter/builders/prod/Linux%20Android%20Debug%20Engine/18769/overview

```
io.flutter.embedding.engine.mutatorsstack.FlutterMutatorViewTest > sendAccessibilityEvents FAILED
    java.lang.ClassCastException: Cannot cast io.flutter.plugin.platform.PlatformViewWrapperTest$ShadowViewGroup to io.flutter.embedding.engine.mutatorsstack.FlutterMutatorViewTest$ShadowViewGroup
        at java.base/java.lang.Class.cast(Class.java:3606)
        at android.view.ViewGroup.requestSendAccessibilityEvent(ViewGroup.java)
        at io.flutter.embedding.engine.mutatorsstack.FlutterMutatorView.requestSendAccessibilityEvent(FlutterMutatorView.java:178)
        at io.flutter.embedding.engine.mutatorsstack.FlutterMutatorViewTest.sendAccessibilityEvents(FlutterMutatorViewTest.java:247)
```

Also seen on:
https://ci.chromium.org/ui/p/flutter/builders/prod/Linux%20Android%20Debug%20Engine/18762/overview
https://ci.chromium.org/ui/p/flutter/builders/prod/Linux%20Android%20Debug%20Engine/18763/overview
https://ci.chromium.org/ui/p/flutter/builders/prod/Linux%20Android%20Debug%20Engine/18764/overview
https://ci.chromium.org/ui/p/flutter/builders/prod/Linux%20Android%20Debug%20Engine/18765/overview
https://ci.chromium.org/ui/p/flutter/builders/prod/Linux%20Android%20Debug%20Engine/18766/overview
https://ci.chromium.org/ui/p/flutter/builders/prod/Linux%20Android%20Debug%20Engine/18767/overview

**Comment by zanderso** [comment]
Maybe related https://github.com/flutter/engine/pull/34272

**Comment by utzcoz** [comment]
Looks like `io.flutter.plugin.platform.PlatformViewWrapperTest$ShadowViewGroup` might need to extend from `io.flutter.embedding.engine.mutatorsstack.FlutterMutatorViewTest$ShadowViewGroup`.

**Comment by utzcoz** [comment]
@zanderso Looks like the flaky is not related to latest Robolectric version from your later PR. Is it possible to bring Robolectric 4.8.1 back again?

**Comment by zanderso** [comment]
> @zanderso Looks like the flaky is not related to latest Robolectric version from your later PR. Is it possible to bring Robolectric 4.8.1 back again?

Yes.

**Comment by utzcoz** [comment]
My network was broken recent days and I can't build Flutter locally now. Could you help to bring it back? Thanks.

**Comment by zanderso** [comment]
The test was removed, so I'll close this.
